CMakeLists.txt

```diff
@@ -208,7 +208,7 @@ else() # non-x86 platforms do not have SSE
     set(SSE2 OFF)
 endif()
 
-set(AUDIOAPI "default" CACHE STRING "Audio API to use (one of {default,coreaudio,jack,portaudio,bela})")
+set(AUDIOAPI "default" CACHE STRING "Audio API to use (one of {default,coreaudio,jack,portaudio,bela|openal|webaudio})")
```

Comment:
Why is `openal` present here? Skimming through the commits, maybe it was considered in the development process, but I don't think it's actually an implemented option.

This also applies to other `CMakeLists.txt` files with the `AUDIOAPI` setting.